extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/WebSocketConnectionImpl.java

```diff
@@ -106,6 +110,14 @@ public Uni<Void> sendPing(Buffer data) {
         return UniHelper.toUni(webSocket.writePing(data));
     }
 
+    void sendAutoPing() {
+        webSocket.writePing(Buffer.buffer("quark")).onComplete(r -> {
```

Comment:
I think we should send `ping` here and not be too distintive. Anything that could identify the technology you are using as a backend can be seen as a potential problem.

Reply:
Hm, the thing is that even `ping` could be used to identify quarkus here because there's no standard payload. But I agree that it's less risky.

Reply:
Done.